pymeasure/instruments/anritsu/anritsuMS464xB.py

```diff
@@ -0,0 +1,318 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+from enum import Enum
+from pymeasure.instruments import Instrument, Channel
+from pymeasure.instruments.validators import (
+    strict_range,
+    strict_discrete_set,
+)
+
+
+class CustomEnum(Enum):
+    """Provides additional methods to IntEnum:
```

Comment:
Wha don't you name it CustomIntEnum to show, that is an IntEnum?

Reply:
Also, shouldn't it inherit from `IntEnum`, not `Enum`?

Reply:
sorry for the misunderstanding. The CustomEnum can be both int or str enums. I changed the docstring instead.